chainer/variable.py

```diff
@@ -499,38 +499,38 @@ def debug_print(self):
 
         msg = """{summary}
 - device: {device}
-- backend: {background}
+- backend: {backend}
 - shape: {shape}
 - dtype: {dtype}
 - statistics: {stats}
 - grad: {grad}"""
 
         stats_msg = 'mean={0:.8f}, std={1:.8f}'
 
-        try:
-            device = self.data.device
-        except AttributeError:
-            device = 'CPU'
-
-        with cuda.get_device_from_array(self.data) as dev:
```

Comment:
Why is this device context removed?
I think this is needed when the `self.data` is on GPU other than the current GPU.

Reply:
Thanks for pointing that out. I was not aware of this requirement. Fixed.